Moderation-Policy.md

```diff
@@ -31,34 +31,33 @@ Any alternative Moderation Policy used for a given repository must be included i
 
 ### Terms
 
-* *Collaborator* refers to any individual with configured write (commit) permissions to any Node.js GitHub organization repository *other than the Moderation Repository*. See [GitHub's access permissions documentation](https://help.github.com/articles/what-are-the-different-access-permissions/) for more information.
-* *TSC* refers to the [Node.js Technical Steering Committee](https://github.com/nodejs/node#tsc-technical-steering-committee).
+* *Collaborator* refers to any individual with configured write (commit) permissions to any Node.js GitHub organization repository *other than the Moderation Repository*. See [GitHub's access permissions documentation][] for more information.
```

Comment:
do people that are in the github org and _only_ have write access to the moderation repository even exist? i'm struggling to understand the relevance of that specification